src/modes/map/map_events.cpp

```diff
@@ -93,30 +96,32 @@ void DialogueEvent::_Start()
 
 bool DialogueEvent::_Update()
 {
-    SpriteDialogue *active_dialogue = MapMode::CurrentInstance()->GetDialogueSupervisor()->GetCurrentDialogue();
-    if((active_dialogue != nullptr) && (active_dialogue->GetDialogueID() == _dialogue_id))
-        return false;
-    else
-        return true;
+    SpriteDialogue *active_dialogue =
+      MapMode::CurrentInstance()->GetDialogueSupervisor()->GetCurrentDialogue();
+    return !(active_dialogue != nullptr &&
+             active_dialogue->GetDialogueID() == _dialogue_id);
 }
 
 // -----------------------------------------------------------------------------
 // ---------- ShopEvent Class Methods
 // -----------------------------------------------------------------------------
 
-ShopEvent* ShopEvent::Create(const std::string& event_id, const std::string& shop_name)
+ShopEvent* ShopEvent::Create(const std::string& event_id,
+                             const std::string& shop_name)
 {
     return new ShopEvent(event_id, shop_name);
 }
 
 void ShopEvent::_Start()
 {
     ShopMode *shop = new ShopMode(_shop_id);
-    for(std::set<std::pair<uint32_t, uint32_t> >::iterator it = _items.begin(); it != _items.end(); ++it)
-        shop->AddItem((*it).first, (*it).second);
+    for(std::set<std::pair<uint32_t, uint32_t> >::iterator it = _items.begin();
+        it != _items.end(); ++it)
```

Comment:
I'm against this change because it puts the for condition end at the same level than the action, which is confusing. Please use braces if doing so.